app/js/disease-associations-controller.js

```diff
@@ -107,7 +86,7 @@ angular.module('cttvControllers')
      * }
      * getFacets(filters);
      */
```

Comment:
Hi Rippa :)
If I understand the question correctly, you'd need to expose the function via the scope, then I'd set a directive attribute to accept a callback function that you can invoke when needed.
So you can either declare the function as a scope variable when you defined it
```
// var getFacets = function(filters){
// changes to
$scope.getFacets = function(filters){
```
or expose it though a new scope var:
`$scope.getFacets = getFacets;`